app/auth.go

```diff
@@ -11,23 +11,63 @@
 //    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 //    See the License for the specific language governing permissions and
 //    limitations under the License.
+
 package app
 
 import (
 	"os"
 	"strings"
 
 	"github.com/mendersoftware/mender/client"
+	"github.com/mendersoftware/mender/conf"
 	"github.com/mendersoftware/mender/datastore"
 	dev "github.com/mendersoftware/mender/device"
 	"github.com/mendersoftware/mender/store"
 	"github.com/pkg/errors"
 	log "github.com/sirupsen/logrus"
 )
 
+// Constants for auth manager request methods
+const (
+	MethodGetAuthToken        = "GET_AUTH_TOKEN"
+	MethodFetchAuthToken      = "FETCH_AUTH_TOKEN"
+	MethodForceFetchAuthToken = "FORCE_FETCH_AUTH_TOKEN"
```

Comment:
The difference is that FORCE_FETCH_AUTH_TOKEN will ignore authorization data already present, while FETCH_AUTH_TOKEN  it will return the auth data stored in the store, or make a new auth request if empty. GET_AUTH_TOKEN never performs API calls, only returns the current valule.

Reply:
But `FETCH_AUTH_TOKEN` is not supposed to return anything. It is an action, "fetch token now". You need to call `GET_AUTH_TOKEN` to get what is actually stored.

Reply:
There is only FETCH now, as agreed.